### DISCUSSION TITLE
Multiple ESP32-S3-Box-3's

### DISCUSSION THREAD

**Discussion by Bytelink5616** [BigBobbas/ESP32-S3-Box3-Custom-ESPHome#105] [General]
I want to add more than one to my system and I know they need unique names but I also read in the instructions that I should not change the name after I add one to Home Assistant ESPHOME.  I know I can edit the file once in ESPHOME but how do I do it before so I have unique names.

**Comment by BigBobbas** [comment]
If you change the name in the config in the ESPHome dashboard, before adding the device to the HA ESPHome integration then that will be fine.
Or if you have already added the device to the integration, then delete it , then rename in the config ->install the config -> add back into the integration

I hope that helps 
Many Thanks 
Rob 👍

**Comment by Bytelink5616** [comment]
Thank you

Get Outlook for Android<https://aka.ms/AAb9ysg>
________________________________

**Comment by Bytelink5616** [comment]
Rob

Can I ask one mor thing. I do not seem to be able to work out how to reset my devices to factory settings.
I have googled a lot and all seem to come up empty and thought you might know.


Ian Robson

Bytelink LLC.

5616 Devon Street

Port Orange, FL 32127

(386) 235-9563  Cell



________________________________

**Comment by fowlerjk** [comment]
Trying this, but when making a new YAML file it will not compile.

**Comment by youkorr** [comment]
the logs

**Reply by fowlerjk** [reply]
Will post once the second try completes or fails

**Reply by fowlerjk** [reply]
```
Compiling .pioenvs/esp32-s3box-front-room/components/esp-tflite-micro/tensorflow/lite/micro/micro_interpreter_graph.cc.o
xtensa-esp32s3-elf-g++: fatal error: Killed signal terminated program cc1plus
compilation terminated.
*** [.pioenvs/esp32-s3box-front-room/src/main.cpp.o] Error 1
========================= [FAILED] Took 583.21 seconds =========================
```

**Reply by BigBobbas** [reply]
Looks like whatever machine you are compiling on doesn't have enough resources. May just need a restart. Or you can run esphome locally

**Reply by fowlerjk** [reply]
OK thank you, I can work with that!

**Reply by fowlerjk** [reply]
Restarted, compiled ok, and seemingly installed ok. But the unit won't light up or turn on. Appears to be bricked.

**Comment by youkorr** [comment]
press the reset button

**Reply by fowlerjk** [reply]
No response

**Comment by youkorr** [comment]
press the reset button and then the boot button, release reset, while holding the reset button, go to esphome, connect, look for your bin file and release the rest button and it is installed

**Comment by youkorr** [comment]
This happens to me often when I create programs but it doesn't work.

**Comment by youkorr** [comment]
does it work?

**Reply by fowlerjk** [reply]
I can get the web interface to show but the unit itself will not respond at all.

**Comment by youkorr** [comment]
is that you find the box when you try to flash with esphome web

**Comment by youkorr** [comment]
![Capture d’écran 2025-02-27 223514](https://github.com/user-attachments/assets/b2d277f4-8d54-405c-8c76-528066de9d5c)

**Reply by fowlerjk** [reply]
Did this multiple times and FINALLY got it to work with an original BIN file.

**Comment by youkorr** [comment]
bizarre

**Comment by youkorr** [comment]
How did you proceed for the new firmware?

**Comment by youkorr** [comment]
![Capture d’écran 2025-02-27 225206](https://github.com/user-attachments/assets/8043662e-dee4-4de4-8cf9-7555ed1acda2)

**Reply by fowlerjk** [reply]
That's what I wound up doing, I had a bin file from my first box that I installed on it.

**Comment by youkorr** [comment]
but what you did delete is the old file that was in the bin

**Comment by youkorr** [comment]
use notepad++
